torch/lib/c10d/reducer.cpp

```diff
@@ -321,7 +321,7 @@ void Reducer::mark_variable_ready_dense(size_t variable_index) {
       // to bucket_view. If grad has already been set as views of buckets in
       // previous iterations, no copy is needed.
       if (!grad.is_alias_of(bucket_view)) {
-        this->copy_grad_to_bucket(grad, bucket_view);
+        bucket_view.copy_(grad);
```

Comment:
where result is going to be divided by divFactor_?

Reply:
The division is done in the implementation of `allreduce_hook`. No need to have another division. Will remove `divFactor_ ` in a separate PR.

Reply:
It will be done in the allreduce comm hook. No need to use a separate `divFactor_`